scripts/output.py

```diff
@@ -0,0 +1,66 @@
+"""
+This script contains functions to get various output variables
+from Cyclus's sqlite output database
+"""
+
+import pandas as pd
+import numpy as np
+import cymetric as cym
+from cymetric import fco_metrics
+
+
+def initialize_df(scenario_index, scenarios_nums):
+    """This function creates a pandas dataframe with scenario_index
+    as index title and scenarios_nums as index values.
+    This is used to initialize a dataframe to add subsequent columns to.
+    INPUT
+    scenario_index: title of index
+    scenario_nums: list of scenario numbers
+    OUTPUT
+    df: initialized pandas dataframe
+    """
+    df = pd.DataFrame(index=scenarios_nums)
+    df.index.name = scenario_index
+    return df
+
+
+# sensitivity
+def sensitivity(base_case, init_df):
+    """ This function takes a dataframe
```

Comment:
The docstring shouldn't start with a space. Consider starting it on the next line. Also, please give some detail. What is the structure of the expected dataframe... ?